pkg/cmd/pulumi/import.go

```diff
@@ -427,9 +428,10 @@ func newImportCmd() *cobra.Command {
 			}
 
 			opts.Engine = engine.UpdateOptions{
-				Parallel:      parallel,
-				Debug:         debug,
-				UseLegacyDiff: useLegacyDiff(),
+				Parallel:         parallel,
+				Debug:            debug,
+				UseLegacyDiff:    useLegacyDiff(),
+				ProtectResources: protectResources,
```

Comment:
Let's add this as a flag on each import record rather than an `UpdateOption`.

Reply:
👍